### PR TITLE
Add FAMBench xlmr model

### PR DESCRIPTION
**Pull request by xuzhao9**
This PR adds the xlmr model from [FAMBench](https://github.com/facebookresearch/FAMBench/blob/main/benchmarks/xlmr/ootb/xlmr.py) to the TorchBench suite.
It uses the same configuration as the original model, but runs only 1 batch.

### CODE REVIEW

**Comment by xuzhao9** [comment]
@erichan1 can I get a review?

File: torchbenchmark/models/fambench_xlmr/__init__.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    from xlmr_parser import init_argparse
+
+from torchbenchmark.util.model import BenchmarkModel
+from torchbenchmark.tasks import NLP
+import torch.nn.functional as F
+
+class Model(BenchmarkModel):
+    task = NLP.LANGUAGE_MODELING
+    FAMBENCH_MODEL = True
+    # original parameters: FAMBench/benchmarks/run_xlmr_ootb.sh
```
Comment (erichan1) [inline torchbenchmark/models/fambench_xlmr/__init__.py:32]: run_xlmr_ootb.sh is a dummy default config that is too small. Use this one. 

./run_xlmr_ootb.sh -c "--inference-only --famconfig=fb-1dev-A --num-batches=100 --batch-size=96 --sequence-length=64 --vocab-size=250000 --half-model --use-gpu --warmup-batches=20"

**Review by erichan1 — APPROVED** [review]
LGTM! Just small change to model configs.
